doc/developer_guide/APIs.rst

```diff
@@ -443,9 +443,36 @@ computation.
 
 A downside of performing redundant computation in the level-1 halo is
 that any fields being read by the kernel must have their level-1 halo
-clean (up-to-date), which can result in halo exchanges. Note that this
-is not the case for the modified field, it does not need its halo to
-be clean.
+clean (up-to-date), which can result in halo exchanges.
+
+This is also the case for a modified field with ``GH_READINC`` access
+as readinc captures a kernel field with a read (into the level-1 halo)
```

Comment:
I would suggest showing ``readinc`` as code (double backticks). I would also suggest rephrasing this sentence slightly: "captures a kernel field whose data is read (into the level-1 halo) and then incremented."

Reply:
Better thanks.